docs/EXAMPLES.md

```diff
@@ -0,0 +1,517 @@
+Examples
+===============================
+
+> Notes:
+>
+> 1. All the examples are using ES6 syntax;
+>
+> 2. All the arguments in the tables are in the order that they must be passed in the method;
+>
+> 3. The `timeoutInMilliseconds` default is 5000 milliseconds;
+>
+> 4. If you pass the `timeoutInMilliseconds` argument, the timeout is changed only in the respective method. Except on the method `setTimeout`, that changes the timeout of all methods;
+>
+> 5. `timeoutInMilliseconds` turns mandatory if you need to change the `errorMessage`, due to arguments order;
+>
+> 6. If the `errorMessage` argument is not provided a default message implemented for this specific method will be displayed instead. We recommend that you use the default message if possible, because they show a clear message.
```

Comment:
What about 'We recommend that you use the default message because it shows a clear message.'

Reply:
Commit made because this review: e59629c